docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to form 
+arugments about the proving 
+scheme. 
+
+These arguments are used to 
+check samples of elements, 
```

Comment:
"These arguments are used to check samples of elements, against the original ones. " This also seems to be vague, it's not clear what you mean by " checking samples of elements against the original ones"